### DISCUSSION TITLE
Retrieving Github Device Code  no response

### DISCUSSION THREAD

**Discussion by zhuxiaoker** [community/community#7498] [Copilot Conversations] [answered]
when login to  github Retrieving Github Device Code  no response

**Comment by jansorg** [comment, marked as answer]
Is this in a JetBrains IDE?
The plugin needs to access github.com to authenticate with your user. Please verify, that the proxy settings of your IDE or system don't prevent the network request.

**Reply by zhuxiaoker** [reply]
thanks a lot, works for me.

**Reply by chengleqi** [reply]
very useful

**Reply by Symbolk** [reply]
Sry but this seems still not work for me. I am behind a Shadowsocks system-global proxy, and tried either “No Proxy” or "Auto-detect" in IDEA, but still no response. Can you help me out?

**Reply by AimsCloudDrive** [reply]
> <img alt="@iCouldDrives" width="30" height="30" src="https://avatars.githubusercontent.com/u/91238850?s=60&amp;v=4">Has the author's problem been resolved?

**Comment by albertmenglongli** [comment]
I'm using PyCharm 2021.3.3 (Professional Edition)
Build #PY-213.7172.26, built on March 16, 2022, still have this issue.

**Reply by Utousagi** [reply]
Maybe change the IDE HTTP Proxy Settings? It works for me((

**Reply by 1908658071** [reply]
oh， Mine also has this problem in pycharm，but in idea and phpstom not this problem

**Reply by yippp** [reply]
The same error is shown in my PyCharm 2021.3.3
